libs/features/components/banners/src/lib/components/elewa-group-team-member/elewa-group-team-member.component.scss

```diff
@@ -0,0 +1,247 @@
+.banner-container {
+    position: relative;
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+}
+  
+  .banner {
+    position: absolute;
+    background-color: var(--elewa-group-website-color-bg);
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+  
+  .banner-image {
+    display:flex;
+    width: 40%;
+    height: 70%;
+    margin-bottom: 3%;
+    height: 100%;
+    background-size: cover;
+    background-position: center;
+    border-bottom-right-radius: 5%;
+  }
+  
+  .banner-info {
+    height: 70%;
+    width: 40%;
+    padding: 0 40px;
+    text-align: left;
+    color: var(--elewa-group-website-color);
+    
+}
+  
+  .banner-info h1 {
+    font-size: var(--elewa-group-website-font-size-subheading);
+    font-family: var(--elewa-group-website-dmsans-regular);
+    margin-bottom: 1px;
+
+  }
+  
+  .banner-info h2 {
+    font-family: var(--elewa-group-website-dmsans-bold);
+    font-size: var(--elewa-group-website-font-size-description);
+    font-size: 15px;
+    margin-bottom: 40px;
+  }
+
+  .banner-info p {
+    color: var(--elewa-group-website-color);
+    font-family: var(--elewa-group-website-dmsans-regular);
+     font-size: var(--elewa-group-website-font-size-title);
+  }
+
+  .linkedin-icon {
+    display: flex;
+    width: 50px;
+    height: 50px;
+    background-color: black;
+    border-radius: 50%;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    bottom: 70px;
+    left: 70px;
+  }
+  
+ 
+  .fa-linkedin-in {   
+    color: white;
+    font-size: 25px;
+}
+
+.banner-controls {
+    position: absolute;
+    bottom: 90px;
+    right: 0px;
+    display: flex;
+    align-items: center;
+    margin-left: 5%;
+    margin-right: 5%;
+    padding: 0 40px;  
+  }
+  
```

Comment:
The navigation controls need to remain on the right side even on mobile view, the scrollbar at the bottom is also too thick on mobile. Make it a bit thinner.

Reply:
fixed --> https://github.com/italanta/elewa-group/pull/132